jibri/rootfs/defaults/jibri.conf

```diff
@@ -0,0 +1,100 @@
+{{ $ENABLE_JIBRI_HTTP_API_PORT := .Env.ENABLE_JIBRI_HTTP_API_PORT | default "1" | toBool }}
+jibri {
+    // A unique identifier for this Jibri
+    // TODO: eventually this will be required with no default
+    id = "jibri-instance-{{ .Env.JIBRI_INSTANCE_ID }}"
+    
+    api {
+        {{ if $ENABLE_JIBRI_HTTP_API_PORT -}}
+        http {
+            {{  if .Env.JIBRI_HTTP_API_EXTERNAL_PORT -}}
+            external-api-port = {{ .Env.JIBRI_HTTP_API_EXTERNAL_PORT }}
+            {{ end -}}
+            {{ if .Env.JIBRI_HTTP_API_INTERNAL_PORT -}}
+            internal-api-port = {{ .Env.JIBRI_HTTP_API_INTERNAL_PORT }}
+            {{ end -}}
+        }
+        {{ end -}}
+        xmpp {
+            environments = [
+                {
+                    // A user-friendly name for this environment
+                    name = "{{ .Env.XMPP_ENV_NAME }}"
+
+                    // A list of XMPP server hosts to which we will connect
+                    xmpp-server-hosts = [
+                        "{{ .Env.XMPP_SERVER }}"
+                    ]
+
+                    // The base XMPP domain
+                    xmpp-domain = "{{ .Env.XMPP_DOMAIN }}"
+
+                    // The MUC we will join to announce our presence for
+                    // recording and streaming services
+                    control-muc {
+                        domain = "{{ .Env.XMPP_INTERNAL_MUC_DOMAIN }}"
+                        room-name = "{{ .Env.JIBRI_BREWERY_MUC }}"
+                        nickname = "jibri-instanse-{{ .Env.JIBRI_INSTANCE_ID }}"
+                    }
+
+                    // The login information for the control MUC
+                    control-login {
+                        domain = "{{ .Env.XMPP_AUTH_DOMAIN }}"
+                        username = "{{ .Env.JIBRI_XMPP_USER }}"
+                        password = "{{ .Env.JIBRI_XMPP_PASSWORD }}"
+                    }
+
+                    sip-control-muc {
```

Comment:
This is for SIP-Jibri right? Let's drop it for now please.

Reply:
Done it 👍